jest.config.ts

```diff
@@ -0,0 +1,17 @@
+import type { Config } from '@jest/types'
+
+const config: Config.InitialOptions = {
+	preset: 'ts-jest',
+	testEnvironment: 'node',
+	moduleNameMapper: {
+		'^(.*)\\.js$': '$1',
+	},
+	coverageDirectory: 'coverage',
+	coverageProvider: 'v8',
+	collectCoverageFrom: ['src/**/*'],
+	clearMocks: false,
```

Comment:
dont we want to start each test fresh?

Reply:
mean clearMocks:true?

Reply:
Yes. Why not?

Reply:
It says clearMocks:false

Reply:
You're absolutely correct. I'm not sure why I did that. Fixed.